interpret/specification.py

```diff
@@ -156,7 +157,7 @@ def create_covariator(specconf, weatherbundle, economicmodel, config=None, quiet
 
     return covariator
         
-def create_curvegen(csvv, covariator, regions, farmer='full', specconf=None, getcsvvcurve=False, diag_infix=""):
+def create_curvegen(csvv, covariator, regions, farmer='full', specconf=None, getcsvvcurve=False, diag_infix="", othermodels={}):
```

Comment:
I wouldn't risk having a mutable default arg. Even if it's not a problem now, I would totally eliminate the possibility of a mutable default bug in the future because they're so nasty. Suggest swapping this for `None` and then assigning a `{}` if is `None` at the head of the function.

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
Done.